### PR TITLE
 standby node err msgs & toast

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Description

updated err msgs for standby node reserve and toast to reflect status change delay

### Changes

- added two err msgs one for the case where standby node is not reserved by user and one when standby node status is reserved by user but status is not up yet
- added a toast for when user reserves standby node to notify user it will take a while till status is up

### Related Issues

#3730

### Tested Scenarios

- manually selected a standby node, reserved node, waited till node status is up 

### Documentation PR

For UI changes, Please provide the Documentation PR on [info_grid](https://github.com/threefoldtech/info_grid)

#### Preliminary Checks:

- [x] Preliminary Checks
  - Does it completely address the issue linked?
  - What about edge cases?
  - Does it meet the specified acceptance criteria?
  - Are there any unintended side effects?
  - Does the PR adhere to the team's coding conventions, style guides, and best practices?
  - Does it integrate well with existing features?
  - Does it impact the overall performance of the application?
  - Are there any bottlenecks or slowdowns?
  - Has it been optimized for efficiency?
  - Has it been adequately tested with unit, integration, and end-to-end tests?
  - Are there any known defects or issues?
  - Is the code well-documented?
  - Are changes to documentation reflected in the code?

#### UI Checks:

- [x] UI Checks
  - If a UI design is provided/ does it follow it?
  - Does every button work?
  - Is the data displayed logical? Is it what you expected?
  - Does every validation work?
  - Does this interface feel intuitive?
  - What about slow network? Offline?
  - What if the gridproxy/graphql/chain is failing?
  - What would a first time user have a hard time navigating here?

#### Code Quality Checks:

- [x] Code Quality Checks
  - Code formatted/linted? Are there unused imports? .. etc
  - Is there redundant/repeated code?
  - Are there conditionals that are always true or always false?
  - Can we write this more concisely?
  - Can we reuse this code? If yes, where?
  - Will the changes be easy to maintain and update in the future?
  - Can this code become too complex to understand for other devs?
  - Can this code cause future integration problems?

### Testing Checklist

- [x] Does it Meet the specified acceptance criteria.
- [x] Test if changes can affect any other functionality.
- [ ] Tested with unit, integration, and end-to-end tests.
- [x] Tested the un-happy path and rollback scenarios.
- [ ] Documentation updated to meet the latest changes.
- [ ] Check automated tests working successfully with the changes.
- [x] Can be covered by automated tests.

### General Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [ ] Code format and docstring
- [x] Screenshots/Video attached (needed for UI changes)

image
image

image
image

### CODE REVIEW

File: packages/playground/src/components/node_selector/TfManualNodeSelector.vue [hunk 1]
```diff
@@ -142,6 +142,11 @@ export default {
         switch (true) {
           case node.status === "down":
             throw `Node ${nodeId} is down`;
+          case node.status === "standby" && node.rentedByTwinId !== gridStore.client.twinId:
```
Comment (maayarosama) [inline packages/playground/src/components/node_selector/TfManualNodeSelector.vue]: We can add an enum for node status instead of using strings directly

**Comment by maayarosama** [comment]
Can't test for lack of standby nodes on dev and qa

**Comment by amiraabouhadid** [comment]
> Can't test for lack of standby nodes on dev and qa

there is plenty on mainnet, dm if you need an account to test with

**Review by ramezsaeed — COMMENTED** [review]
Test Scenario:
- check when try to deploy on a standby node (which not reserved); should fail with a proper err msg (can't deploy on a standby node unless you reserve it)

**Comment by amiraabouhadid** [comment]
> Test Scenario:
> 
> * check when try to deploy on a standby node (which not reserved); should fail with a proper err msg (can't deploy on a standby node unless you reserve it)

Tested that, works, reviewers please confirm too

**Review by samaradel — APPROVED** [review]

**Review by maayarosama — APPROVED** [review]

**Pull request merged**